client/src/test/java/com/sinch/sdk/domains/voice/models/requests/CalloutRequestParametersCustomTest.java

```diff
@@ -18,12 +25,25 @@ public class CalloutRequestParametersCustomTest {
           .setDtfm(DualToneMultiFrequency.valueOf("w123#"))
           .setMaxDuration(32)
           .setIce(
-              "{\"action\": {\"name\": \"connectPstn\", \"number\": \"+12233445566\", \"cli\":"
-                  + " \"+12234325234\", \"amd\": {\"enabled\": \"true\"}}}")
+              SVAMLControl.builder()
+                  .setAction(
+                      ActionConnectPstn.builder()
+                          .setCli("+12234325234")
+                          .setNumber(E164PhoneNumber.valueOf("+12233445566"))
+                          .setAnsweringMachineDetectionEnabled(
+                              AnsweringMachineDetection.builder().setEnabled(true).build())
```

Comment:
I understand the logic to stick to the JSON mapping, but for this particular case, maybe a helper method would be more friendly like `setAnsweringMachineDetectionEnabled(true)` which would act like a DSL

Reply:
I've asked myself same question and it was my 1st implementation ..... :-)
(For Conversation API there is also same kind of extra level depth with only a single field)
For consistency I have kept current builder and extra level to prevent future potential new fields
We can imagine having to function with different parameter signature to provide a "quick access" to the setting in the future; in this case, always for consistency purpose across domains, it should be addressed at "larger scale"